src/azul/service/manifest_service.py

```diff
@@ -512,14 +515,11 @@ def command_lines(self,
                       url: str
                       ) -> Optional[JSON]:
         if manifest is None:
-            return ManifestGenerator.command_lines(url)
+            generator_cls = ManifestGenerator
         else:
-            generator = ManifestGenerator.cls_for_format[manifest.format_]
-            if generator.use_content_disposition_file_name:
-                file_name = manifest.object_key.rpartition('/')[2]
-            else:
-                file_name = manifest.file_name
-            return generator.command_lines(url, file_name)
+            generator_cls = ManifestGenerator.cls_for_format[manifest.format_]
```

Comment:
Rename ManifestGenerator.cls_for_format to ManifestGenerator._cls_for_format and encapsulate it in a method so that

```suggestion
            format = None if manifest is None else manifest.format_
            generator_cls = ManifestGenerator.cls_for_format(format)
```